pom.xml

```diff
@@ -91,7 +91,7 @@
         <quickcheck.version>0.7</quickcheck.version>
         <httpclient.version>4.5.3</httpclient.version>
         <findbugs.version>3.0.1</findbugs.version>
-        <redis.version>2.6.0</redis.version>
+        <redis.version>2.9.0</redis.version>
```

Comment:
This is the latest version: https://mvnrepository.com/artifact/redis.clients/jedis

I don't think there is a direct correlation between jedis and redis versions.